### ISSUE TITLE
doing things we might not want it to do

### ISSUE THREAD

**Issue by wseery** [AnthonyOSX/GO-Bot#59]
I just tried the new version and immediately, it started evolving all my stuff.  I wanted to save some of that, I definitely didn't want to evolve my metapod, etc.  I would like to see more customizeable things like the use of different pokeballs, specifying which pokemon to transfer, etc.  I will definitely be using version 3 because of the evolve thing.

**Comment by wseery** [comment]
@AnthonyOSX please let us opt out of some of the things the bot does, the transferring of specific pokemon and not evolving is really high priority in my opinion

**Comment by discordmaru** [comment]
Yeah a way to click a list of what to evolve when enough candies. A way to keep more than 1 of each top cp (example 3 top cp eevees). Inventory management, and I just saw new release use a great ball on a darn rat. So a way to customize what level or type of pokemon so good poke balls aren't wasted on junky things.

**Comment by dgol1115** [comment]
honestly i think just the ability to turn off some features (transfer, evolve) would be best. we cant pick and choose too much when a robot is playing the game for us
